project/plugins.sbt

```diff
@@ -14,3 +14,6 @@ addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.8.2")
 addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.0.0")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.2.0")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.2.2")
+
+libraryDependencies += "org.scala-sbt" % "scripted-plugin" % sbtVersion.value
```

Comment:
am i correct that this is separate from the fix and enables testing of plugins? 

if so, can we split this into to two separate PRs?